setup.py

```diff
@@ -53,10 +52,11 @@
     ],
     include_package_data=True,
     zip_safe=False,
-    python_requires=">=3.7",
+    python_requires=">=3.8",
     install_requires=[
         # BEGIN requirements
-        "Django>=2.2",
+        "Django>=4.2",
+        "setuptools",
```

Comment:
Why is `setuptools` needed as a _runtime_ dependency? This will install `setuptools` in end user environments.

This should only be a _build time_ dependency, right? I don't see any code changes where `django-downloadview` actually `import`s `setuptools` at runtime.

Reply:
A better long term solution would be to migrate from `setup.py` to `pyproject.toml` as mentioned here

- https://packaging.python.org/en/latest/guides/writing-pyproject-toml/

Then this build dependency would be properly expressed as 

```toml
[build-system]
requires = ["setuptools >= 61.0"]
build-backend = "setuptools.build_meta"
```

Reply:
`pkg_resources` is part of `setuptools`, so there is a runtime dependency; it's just not obvious. I described this in #208, but added it here since I was already touching setup.py